packages/cli-kit/src/private/node/ui/components/Tasks.tsx

```diff
@@ -25,6 +27,7 @@ interface TasksProps<TContext> {
   silent?: boolean
   onComplete?: (ctx: TContext) => void
   abortSignal?: AbortSignal
+  noColor?: boolean
```

Comment:
I only introduced this to inject `true` in tests because mocking `output.js` (where we get `shouldDisplayColors`) led to very weird results. It's a private interface, so I don't think this should matter all that much.